Zend/tests/return_types/033.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+__set can only declare void return
+--FILE--
+<?php
+
+class Foo {
+	function __set($name, $value) : string {}
+}
+?>
+--EXPECTF--
+Fatal error: Return type of Foo::__set() must be void when declared in %s on line %d
```

Comment:
Maybe, following the format of non-void return value error message would be better:
`Fatal error: Foo::__set(): Return type must be void when declared in %s on line %d`